lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,108 @@
+// Import partials here
+@import 'base.normalize';
+@import 'settings.variables';
+@import 'mixins';
+
+// center main tag in the middile
+#app {
```

Comment:
Excellent job with the detailed comments. Makes reading your code very seamless.